app/importers/concerns/importable.rb

```diff
@@ -1,10 +1,16 @@
-module Importer
+module Importable
+  extend ActiveSupport::Concern
   # The module provides functionality useful for importing source data, and
   # can be included into any class that will do so.
 
-  def self.included(base)
-    base.class_eval do
-      base.send(:prepend, ParentInstanceMethods)
+  included do
+    send(:prepend, ParentInstanceMethods)
+
+    class << self
+      attr_accessor :disabled
+      def disabled?
+        !!disabled
```

Comment:
Awesome, I just needed to know why so I can decide what to do next time I encounter a similar situation. and i like double-bang :+1: